### PR TITLE
Feature/info confirmation screen

### PR DESCRIPTION
**Pull request by FrogTravel**
Hints for "Bond more", "Unstake" and "Staking" confirmation screens 

And +1 screen where I now cannot get due to lack of wnd

### CODE REVIEW

**Comment by antonkhvorov** [comment]
> And +1 screen where I now cannot get due to lack of wnd

You can always either use faucet or ask fellow teammates to share some WND :D

File: feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/nominations/ConfirmNominationsFragment.kt [hunk 1]
```diff
@@ -35,6 +35,8 @@ class ConfirmNominationsFragment : BaseFragment<ConfirmNominationsViewModel>(),
         confirmNominationsToolbar.setHomeButtonListener {
             viewModel.backClicked()
         }
+
+        println("--------- CONFIRM NOMINATIONS FRAGMENT")
```
Comment (FrogTravel) [inline feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/nominations/ConfirmNominationsFragment.kt:39]: debug println!

File: feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/ConfirmStakingFragment.kt [hunk 2]
```diff
@@ -117,5 +107,13 @@ class ConfirmStakingFragment : BaseFragment<ConfirmStakingViewModel>() {
...
             bondedAmount?.let { confirmStakeAmount.amountInput.setText(it.toString()) }
         }
+
+        viewModel.unstakingTime.observe {
+            confirmStakingUnstakingPeriodLength.text = getString(R.string.staking_hint_unstake_format, it)
+        }
+
+        viewModel.eraHoursLength.observe {
+            confirmStakingEachEraLength.text = getString(R.string.staking_hint_rewards_format, it)
+        }
```
Comment (valentunn) [inline feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/ConfirmStakingFragment.kt:117]: We commonly format strings in ViewModel itself (do in background as much work as possible + unification with other parts of project)

File: feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/ConfirmStakingViewModel.kt [hunk 3]
```diff
@@ -120,6 +121,16 @@ class ConfirmStakingViewModel(
...
 
+    val unstakingTime = liveData(Dispatchers.Default) {
+        val lockupPeriod = interactor.getLockupPeriodInDays()
+        emit(resourceManager.getQuantityString(R.plurals.staking_main_lockup_period_value, lockupPeriod, lockupPeriod))
+    }
+
+    val eraHoursLength = liveData(Dispatchers.Default) {
+        val hours = interactor.getEra1HoursLength()
+        emit(resourceManager.getQuantityString(R.plurals.common_hours_format, hours, hours))
+    }
```
Comment (valentunn) [inline feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/ConfirmStakingViewModel.kt:132]: Afaik we dicided to not to use livedata but flow instead (replacement is `flow { emit() }.share()`)

**Review by valentunn — APPROVED** [review]

File: feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/ConfirmStakingViewModel.kt (hunk 3 above)
Comment (FrogTravel) [reply feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/ConfirmStakingViewModel.kt]: Done

File: feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/ConfirmStakingFragment.kt (hunk 2 above)
Comment (FrogTravel) [reply feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/ConfirmStakingFragment.kt]: Done

File: feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/domain/StakingInteractor.kt [hunk 4]
```diff
@@ -160,6 +162,13 @@ class StakingInteractor(
 
     suspend fun getLockupPeriodInDays() = getLockupPeriodInDays(getSelectedNetworkType())
 
+    suspend fun getEra1HoursLength(): Int {
```
[the remark this answers is not in the document]
Comment (FrogTravel) [reply feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/domain/StakingInteractor.kt]: Done

**Pull request merged**